app/tournamentBusiness/tournamentProgress.js

```diff
@@ -0,0 +1,121 @@
+import $ from "jquery";
+import * as tournamentAlerts from "./tournamentAlerts";
+
+var tournament;
+
+export function init(t) {
+    tournament = t;
+
+    generateHTMLTournamentBracket();
+}
+
+function generateHTMLTournamentBracket() {
+    cleanBracket();
+    divideBracketVerticallyInRounds(tournament.getRounds());
+    appendAllMatchesWithSpacesToHtml(tournament.matches);
+    bindClickEventOnWinButtons(tournament.matches);
+}
+
+function cleanBracket() {
+    $("#tournamentBracket").empty();
+}
+
+function divideBracketVerticallyInRounds(rounds) {
+    var colSize = 100/rounds;
+
+    for (var r = 1; r <= rounds; r++) {
+        $("#tournamentBracket").append(
+            `<div class="col-xs-1" id="round${r}" style="width: ${colSize}%; padding: 0;"></div>`
+        );
+    }
+}
+
+function appendAllMatchesWithSpacesToHtml() {
+    tournament.matches.forEach( function(match, i, matches) {
+        if (match.round > 1) {
+            appendTransparenSpacesToHtml(match.round, matches[i-1].round);
+        }
+        appendMatchHtmlPanel(match, i);
+    });
+}
+
+function appendTransparenSpacesToHtml(round, lastRound) {
+    $(`#round${round}`).append(
```

Comment:
These IDs make the code also fragile. I think you could avoid them by keeping the elements when they are generated.